src/js/post-911-gib-status/components/InfoPair.jsx

```diff
@@ -4,9 +4,16 @@ import PropTypes from 'prop-types';
 
 class InfoPair extends React.Component {
   render() {
+    const { spacingClass } = this.props;
+
+    let gridRowClasses = 'usa-grid-full';
```

Comment:
yes, way better! you always give good advice on consolidating lines of code :) will make that change!